crates/weaver_resolver/src/registry.rs

```diff
@@ -329,22 +278,15 @@ fn group_from_spec(group: GroupSpecWithProvenance) -> UnresolvedGroup {
 fn resolve_attribute_references(
     ureg: &mut UnresolvedRegistry,
     attr_catalog: &mut AttributeCatalog,
-) -> bool {
+) -> Result<(), Error> {
     loop {
-        let mut unresolved_attr_count = 0;
+        let mut errors = vec![];
         let mut resolved_attr_count = 0;
 
         // Iterate over all groups and resolve the attributes.
         for unresolved_group in ureg.groups.iter_mut() {
             let mut resolved_attr = vec![];
 
-            if unresolved_group.group.extends.is_some() {
-                // If the group has an `extends` clause, we need to resolve the
-                // `extends` references first.
-                unresolved_attr_count += unresolved_group.attributes.len();
```

Comment:
It seems to me that's not needed, as extends are supposed to be resolved before this function is called?

Reply:
I agree. The recent changes to the resolution steps have made this step unnecessary. Thanks for pointing it out.